posthog/clickhouse/property_groups.py

```diff
@@ -210,12 +210,24 @@ def get_alter_modify_statements(
         },
         "logs": {
             "attributes": {
-                "custom": PropertyGroupDefinition(
-                    "true",
-                    lambda key: True,
+                "str": PropertyGroupDefinition(
```

Comment:
Agreed this could probably be cleaner or handled internally by HogQL but happy to go with it if it unblocks you for now.

Is there any way we could return the format of the property from the values query and use that instead?